torchaudio/models/_wavernn.py

```diff
@@ -192,3 +193,125 @@ def forward(self, specgram: Tensor) -> Tensor:
         upsampling_output = upsampling_output.squeeze(1)[:, :, self.indent:-self.indent]
 
         return upsampling_output, resnet_output
+
+
+class _WaveRNN(nn.Module):
+    r"""WaveRNN model based on "Efficient Neural Audio Synthesis".
+
+    The paper link is https://arxiv.org/pdf/1802.08435.pdf. The input channels of waveform
+    and spectrogram have to be 1. The product of upsample_scales must equal hop_length.
```

Comment:
nit: The product of ``upsample_scales`` must equal ``hop_length``.
(with double backticks)

Reply:
Thanks. Updated.